### PR TITLE
[14.0][ADD] sale_partner_shipping_invoice_domain: add to 14.0

### PR DESCRIPTION
**Pull request by kevinkhao**
Adds a domain on shipping address to filter out the irrelevant contacts instead of choosing from every contact in DB

Screenshot from 2021-01-26 10-56-59

### CODE REVIEW

**Comment by sebastienbeau** [comment]
I think it will be simplier to depend of sale_commercial_partner : 
https://github.com/OCA/sale-workflow/tree/14.0/sale_commercial_partner 

Then you can just add a domain on invoice and delivery address that will be [("id", "child_of", commercial_partner_id)]

**Comment by hparfr** [comment]
Can you make a pr against sale-workflow ?

File: sale_partner_shipping_invoice_domain/views/sale_order.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        <field name="arch" type="xml">
+            <field name="partner_shipping_id" position="attributes">
+                <attribute
+                        name="domain"
+                >[('id', 'child_of', commercial_partner_id)]</attribute>
+            </field>
+            <field name="partner_invoice_id" position="attributes">
+                <attribute
+                        name="domain"
+                >[('id', 'child_of', commercial_partner_id)]</attribute>
```
Comment (hparfr) [inline sale_partner_shipping_invoice_domain/views/sale_order.xml]: why not set this domain in the (python) field instead ?

Comment (sebastienbeau) [reply sale_partner_shipping_invoice_domain/views/sale_order.xml]: +1

Comment (kevinkhao) [reply sale_partner_shipping_invoice_domain/views/sale_order.xml]: done

**Comment by kevinkhao** [comment]
> Can you make a pr against sale-workflow ?

https://github.com/OCA/sale-workflow/pull/1433

**Comment by kevinkhao** [comment]
@hparfr should i close this one here ?

**Comment by hparfr** [comment]
Yes thanks

**Pull request closed without merging**